### PR TITLE
Enhancement: make icon preview "sticky" functional

### PR DESCRIPTION
**Pull request by sandsng**
Noticed the icon preview was not "sticky" as desired. This PR updates the CSS so that it properly scrolls with the user.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-website/src/components/icons-list/IconsList.tsx [hunk 1]
```diff
@@ -16,7 +16,6 @@ import type {IconObject, IconComponent, IconsListProps, GroupedList} from './typ
 import {IconCard} from './IconCard';
 import {SiteLink} from '../SiteLink';
 import {IconListItem} from './IconListItem';
-import {STICKY_COLUMN_OFFSET} from '../../constants';
```
Comment (TheSisb) [inline packages/paste-website/src/components/icons-list/IconsList.tsx:19]: suggestion: can you delete this constant from the constants file as well? Doesn't seem to be used elsewhere

**Review by TheSisb — COMMENTED** [review]
Such a nice improvement, thank you!

**Review by TheSisb — APPROVED** [review]

**Review by nkrantz — APPROVED** [review]
Such a good improvement! Nice job

**Review by gloriliale — APPROVED** [review]
yes pls.

**Pull request merged**